src/field/FieldTrackView.hh

```diff
@@ -0,0 +1,95 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file FieldTrackView.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/Types.hh"
+#include "base/Macros.hh"
+#include "physics/base/Units.hh"
+#include "field/FieldInterface.hh"
+#include "geometry/GeoTrackView.hh"
+#include "physics/base/ParticleTrackView.hh"
+
+#include <VecGeom/navigation/VNavigator.h>
+#include <VecGeom/navigation/NavigationState.h>
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Read/write view to the field state of a single track.
+ */
+
+class FieldTrackView
+{
+  public:
+    //@{
+    //! Type aliases
+    using NaviState = vecgeom::NavigationState;
+    using Navigator = vecgeom::VNavigator;
+    //@}
+
+  public:
+    // Construct from GeoTrackView and ParticleTrackView
+    inline CELER_FUNCTION FieldTrackView(const GeoTrackView&      gt_view,
+                                         const ParticleTrackView& pt_view);
+
+    // Copy Construct
+    inline CELER_FUNCTION FieldTrackView(const FieldTrackView& other) = default;
```

Comment:
This shouldn't be necessary; a force-defaulted copy constructor should only ever be require if you define a custom move constructor.